css/style.css

```diff
@@ -219,6 +184,29 @@ footer p {
     margin-bottom: 100px;
 }
 
+
+/* General Styling */
+
+.jus-cen {
```

Comment:
I'd suggest avoiding abbreviations in variable names. You can read a bit more [here](https://syllabus.codeyourfuture.io/guides/code-style-guide#avoid-short-names) about avoiding short names.

Reply:
Thanks, I didn't know so I have changed them now